lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -882,9 +882,11 @@ class SemanticsObject {
     }
 
     if (!effectiveTransformIsIdentity) {
+      final Position2D position2d =
```

Comment:
suggestion: translate(0, 0) and use the resulting offset, instead of adding Position2D and adding matrix utils.

Reply:
I'm not sure how `translate(0, 0)` helps here. But could the PR be simplified to this?
```dart
final Vector3 translation = effectiveTransform.getTranslation();
element.style
  ..top = '${translation.y}px'
  ..left = '${translation.x}px';
```
or:
```dart
final Matrix4 translationOnly =
    Matrix4.identity()
    ..setTranslation(effectiveTransform.getTranslation());
element.style
    ..transformOrigin = '0 0 0'
    ..transform = matrix4ToCssTransform(translationOnly);
```

Reply:
Both options does not work unfortunately.
a) Boundary does not get correctly recognized by the screen reader
```
..transformOrigin = '0 0 0'
    ..transform = matrix4ToCssTransform(translationOnly);
```
b) Loses other information.
```
element.style
  ..top = '${translation.y}px'
  ..left = '${translation.x}px';
```

I'll probably still go with option 2 though, since, transforms in 3D are a rare case and now most of the common cases are not working as intended (shown in the issue).

Reply:
@ferhatb I used the method you sent yesterday for computing the bounding rectangle. Thanks!